core/src/test/java/org/elasticsearch/index/query/FieldMaskingSpanQueryBuilderTest.java

```diff
@@ -94,5 +94,19 @@ public void testValidate() {
         }
         FieldMaskingSpanQueryBuilder queryBuilder = new FieldMaskingSpanQueryBuilder(spanQueryBuilder, fieldName);
         assertValidate(queryBuilder, totalExpectedErrors);
+
+        try {
+            queryBuilder = new FieldMaskingSpanQueryBuilder(new SpanTermQueryBuilder("name", "value"), null);
+            fail("Exception expected");
+        } catch (NullPointerException e) {
```

Comment:
maybe move out to separate test methods and use junit expected exception rather than an empty catch?